src/cgame/cg_debriefing.c

```diff
@@ -3948,6 +3970,7 @@ const char *awardNames[NUM_ENDGAME_AWARDS] =
 	"Highest Experience Points",
 	"Most Highly Decorated",
 	"Highest Fragger",
+	"Highest Assist",
```

Comment:
`Most Assists` is more fitting here imo, just like you have `Most Deaths`.

```suggestion
	"Most Assists",
```

There's also perhaps a conversation to be had here whether this should just be the number of assists, or most damage via assists, as the assists reward XP based on the amount of damage dealt, but that would require quite a bit of rework, so perhaps something to revisit in the future after some playtesting and feedback.